homeassistant/components/homewizard/quality_scale.yaml

```diff
@@ -46,11 +46,7 @@ rules:
   # Gold
   devices: done
   diagnostics: done
-  discovery-update-info:
-    status: todo
-    comment: |
-      The integration doesn't update the device info based on DHCP discovery
-      of known existing devices.
+  discovery-update-info: done
```

Comment:
Not sure about this one, this integration does not DHCP discovery so that would be a whole new feature if even possible (I think it is). But for the existing Zeroconf it already works as expected.

Reply:
```suggestion
  discovery-update-info:
    status: todo
    comment: |
      The integration doesn't update the device info based on DHCP discovery
      of known existing devices.
```